src/main/java/racingcar/domain/Cars.java

```diff
@@ -2,30 +2,46 @@
 
 import java.util.List;
 import java.util.stream.Collectors;
+import racingcar.domain.movable.MovableStrategy;
+import racingcar.utils.Parser;
 
 public class Cars {
 
     private final List<Car> cars;
 
-    public Cars(final List<Car> cars) {
+    private Cars(final List<Car> cars) {
         this.cars = cars;
     }
 
+    public static Cars fromString (final String carNames) {
```

Comment:
`opt + cmd + L`을 이용해서 포맷팅을 해보시면 어떨까요?

```suggestion
    public static Cars fromString(final String carNames) {
```